R/aaa-auto.R

```diff
@@ -3865,3 +3864,15 @@ vertex_path_from_edge_path_impl <- function(graph, start, edge.path, mode=c("out
   res
 }
 
+version_impl <- function() {
+  # Argument checks
+
+
+  on.exit( .Call(R_igraph_finalizer) )
+  # Function call
+  res <- .Call(R_igraph_version, )
```

Comment:
how would I tell Stimulus not to add an empty argument here?

Reply:
I'd appreciate any pointer @Antonov548 @krlmlr (or we can keep this for the hackathon)

Reply:
I thought I have fixed this some time ago in stimulus. 
I will take a look

Reply:
that is something I tried to add. I think I got the YAML tweaks wrong!

Reply:
I do not understand why after rerunning the docker command, this problem isn't fixed. This branch has been rebased on main. Is there something else to do for build-requirements to be followed @Antonov548?